src/libs/ReportUtils.js

```diff
@@ -1159,6 +1159,7 @@ function buildOptimisticIOUReportAction(type, amount, currency, comment, partici
     // IOUs of type split only exist in group DMs and those don't have an iouReport so we need to delete the IOUReportID key
     if (type === CONST.IOU.REPORT_ACTION_TYPE.SPLIT) {
         delete originalMessage.IOUReportID;
+        originalMessage.participants = [currentUserEmail, ..._.pluck(participants, 'login')];
```

Comment:
`participants` was just removed from this function's params in this PR: https://github.com/Expensify/App/pull/18215/files#r1189768768

let's discuss how to resolve here: https://expensify.slack.com/archives/C01GTK53T8Q/p1683719211068359